recipes/rockhound/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "rockhound" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 22b6c90a5dc0e3c37aa2f899b850ddfd18b2c87c2b9d61632e6297870c11a4dc
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - pooch>=0.5
+    - xarray
+    - pandas
+    - rasterio
+
+test:
+  imports:
+    - rockhound
+
+about:
+  home: http://github.com/fatiando/rockhound
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: "Download geophysical models/datasets and load them in Python"
+  description: |
+      RockHound is a Python library to download geophysical models and datasets (PREM,
+      CRUST1.0, ETOPO1) and load them into Python data structures (pandas, numpy,
+      xarray). Many of these models use non-conventional file formats or can be tricky
+      to find on the internet. RockHound knows how to download them if you don't
+      already have them locally, read the file format, and return a nicely formatted
+      data structure. Under the hood, it uses Pooch to manage the downloads.
+  doc_url: https://www.fatiando.org/rockhound
+  dev_url: https://github.com/fatiando/rockhound
+
+extra:
+  recipe-maintainers:
+    - leouieda
```

Comment:
@leouieda Are you happy to be listed as a maintainer of this recipe?

Reply:
@chrisburr yep 👍 Sorry, I should have left a comment here about this from the start.